openscm/adapters/dice.py

```diff
@@ -0,0 +1,303 @@
+            "M_atm_lower": (10, "GtC/a"),
+            # Initial concentration in lower strata 2010 (GtC)
+            "M_l0": (10010, "GtC/a"),
+            # Equilibrium concentration in lower strata (GtC)
+            "M_l_eq": (10000, "GtC/a"),
+            # Lower bound for concentration in lower strata 2010 (GtC)
+            "M_l_lower": (1000, "GtC/a"),
+            # Initial concentration in upper strata 2010 (GtC)
+            "M_u0": (1527, "GtC/a"),
+            # Equilibrium concentration in upper strata (GtC)
+            "M_u_eq": (1350, "GtC/a"),
+            # Lower bound for concentration in upper strata 2010 (GtC)
+            "M_u_lower": (100, "GtC/a"),
+            # Initial atmospheric temp change (°C from 1900)
+            "T_atm0": (0.8, "°C"),
+            # Upper bound for atmospheric temp change (°C from 1900)
+            "T_atm_upper": (40, "°C"),
+            # Initial lower stratum temp change (°C from 1900)
+            "T_ocean0": (0.0068, "°C"),
+            # Lower bound for lower stratum temp change (°C from 1900)
+            "T_ocean_lower": (-1, "°C"),
+            # Upper bound for lower stratum temp change (°C from 1900)
+            "T_ocean_upper": (20, "°C"),
+            # Carbon cycle transition matrix
+            "b12": (0.0181, ""),
+            # Carbon cycle transition matrix
+            "b23": (0.00071, ""),
+            # Climate equation coefficient for upper level
+            "c1": (0.0222, "°C*m^2/W"),
+            # Transfer coefficient upper to lower stratum
+            "c3": (0.09175, "W/m^2/°C"),
+            # Transfer coefficient for lower level
+            "c4": (0.00487, ""),
+            # Forcings of equilibrium CO2 doubling (Wm-2)
+            "fco22x": (3.8, "W/m^2"),
+            # 2010 forcings of non-CO2 GHG (Wm-2)
+            "fex0": (0.25, "W/m^2"),
+            # 2100 forcings of non-CO2 GHG (Wm-2)
+            "fex1": (0.7, "W/m^2"),
+            # Equilibrium temp impact (°C per doubling CO2)
+            "t2xco2": (2.9, "°C"),
+        }
+
+        for n, v in model_parameter_defaults.items():
+            default, unit = v
+            setattr(
+                self,
+                "_{}".format(n),
+                self._parameterset.get_scalar_view(("DICE", n), (), unit).get(default),
+            )
+
+        # Carbon cycle transition matrix
+        self._b11 = 1 - self._b12
+        self._b21 = self._b12 * self._M_atm_eq / self._M_u_eq
+        self._b22 = 1 - self._b21 - self._b23
+        self._b32 = self._b23 * self._M_u_eq / self._M_l_eq
+        self._b33 = 1 - self._b32
+
+        # Total CO2 emissions (GtCO2 per year)
+        self._E = self._parameterset.get_writable_timeseries_view(
+            ("Emissions", "CO2"), (), "GtCO2/a", self._start_time, self._period_length
+        )
+
+        # Concentration in atmosphere (GtC)
+        self._M_atm = self._parameterset.get_writable_timeseries_view(
+            ("Concentration", "Atmosphere"),
```

Comment:
```suggestion
            ("Pool", "CO2", "Atmosphere"),
```

'Pool' isn't the greatest term but might be ok (perhaps 'change in burden' throughout would be better?). In any case, the units give away that this isn't a concentration.

Reply:
Waiting for #22